cloudevents/adapters/github.md

```diff
@@ -670,3 +1030,47 @@ based on the specified event.
 | `time`                | Current time                          |
 | `data`                | Content of HTTP request body          |
 
+### WorkflowDispatchEvent
+
+| CloudEvents Attribute | Value                                 |
+| :-------------------- | :------------------------------------ |
+| `id`                  | "X-GitHub-Delivery" HTTP header value |
+| `source`              | "repository.url" value                |
+| `specversion`         | `1.0`                                 |
+| `type`                | `com.github.workflow_dispatch`        |
+| `datacontentencoding` | Omit                                  |
+| `datacontenttype`     | `application/json`                    |
+| `dataschema`          | Omit                                  |
+| `subject`             | "workflow" value                      |
+| `time`                | Current time                          |
+| `data`                | Content of HTTP request body          |
+
+### WorkflowJobEvent
+
+| CloudEvents Attribute | Value                                       |
+| :-------------------- | :------------------------------------------ |
+| `id`                  | "X-GitHub-Delivery" HTTP header value       |
+| `source`              | "repository.url" value                      |
+| `specversion`         | `1.0`                                       |
+| `type`                | `com.github.workflow_job.` + "action" value |
+| `datacontentencoding` | Omit                                        |
+| `datacontenttype`     | `application/json`                          |
+| `dataschema`          | Omit                                        |
+| `subject`             | "workflow_job.name" value                   |
+| `time`                | Current time                                |
+| `data`                | Content of HTTP request body                |
+
+### WorkflowRunEvent
+
+| CloudEvents Attribute | Value                                       |
+| :-------------------- | :------------------------------------------ |
+| `id`                  | "X-GitHub-Delivery" HTTP header value       |
+| `source`              | "repository.url" value                      |
+| `specversion`         | `1.0`                                       |
+| `type`                | `com.github.workflow_run.` + "action" value |
+| `datacontentencoding` | Omit                                        |
+| `datacontenttype`     | `application/json`                          |
+| `dataschema`          | Omit                                        |
+| `subject`             | "workflow.name" value                       |
```

Comment:
workflow_run.id?

Reply:
I can't figure out of "name" need to be unique or not. I'm assuming 'id' must be.
From co-pilot:
the name field in a GitHub workflow does not need to be unique. The name is used for display purposes and helps you identify the workflow in the GitHub Actions tab

Reply:
actually, should this be the workflow.url ?